idaes/models_extra/co2_capture_and_utilization/unit_models/membrane_1d.py

```diff
@@ -0,0 +1,341 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
```

Comment:
Pylint is reporting a number of issues with this file that should be addressed (it seems GitHub has added a new integration that shows these during code reviews).